.circleci/config.yml

```diff
@@ -29,6 +28,7 @@ jobs:
       - run:
           name: run tests
           command: |
+            rm -rf example
```

Comment:
What is this needed for?

Reply:
to be consistent with https://github.com/yahoojapan/authorization-proxy/blob/master/.circleci/config.yml#L32

I think previously with we provide example code in `/example`.
Remove in all repo if we don't need it.

Reply:
ok, understand

Reply:
may be better to remove it?

Reply:
I think it's fine as it is now. No problem.